src/Things/Structures/Structure.cpp

```diff
@@ -105,7 +105,7 @@ void Structure::enable()
 	}
 
 	sprite().resume();
-	sprite().color(255, 255, 255, 255);
+	sprite().color(NAS2D::Color{255, 255, 255, 255});
```

Comment:
Can also use `NAS2D::Color::White` here.

Reply:
Good point. It would be nice to switch to named constants. Will probably need to do another pass over the code to handle cases like that.